specs/protocol/isthmus/overview.md

```diff
@@ -15,11 +15,12 @@ This document is not finalized and should be considered experimental.
 ## Execution Layer
 
 - [Pectra](https://eips.ethereum.org/EIPS/eip-7600) (Execution Layer):
-  - [EIP-7702](https://eips.ethereum.org/EIPS/eip-7702)
   - [EIP-2935](https://eips.ethereum.org/EIPS/eip-2935)
+  - [EIP-7002](https://eips.ethereum.org/EIPS/eip-7002)
```

Comment:
Did you mean to add this twice?

Reply:
I think you're confusing 7**7**02 and 7**0**02, easy mistake to make

Reply:
ah good eyes